examples/diff/index.html

```diff
@@ -0,0 +1,141 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <!-- Core extensions CSS from Contentful. -->
+  <link rel="stylesheet"
+        href="//contentful.github.io/ui-extensions-sdk/cf-widget-api.css">
+  <!-- Styles for added / removed text -->
+  <style>
+    .diff-text .added {
+      font-weight: bold;
+      color: #5cb85c;
+    }
+    .diff-text .removed {
+      text-decoration: line-through;
+      font-weight: bold;
+      color: #950B02;
+    }
+  </style>
+</head>
+<body>
+
+<!-- Extension DOM Elements -->
+<div class="cf-form-field">
+  <input type="text" class="cf-form-input draft-text">
+  <div class="cf-form-hint">
+    <span class="diff-text"></span>
+  </div>
+</div>
+
+<!-- Contenful UI Extensions SDK API -->
+<script src="//contentful.github.io/ui-extensions-sdk/cf-widget-api.js"></script>
+<!-- Diff Lib -->
+<script src="//cdnjs.cloudflare.com/ajax/libs/jsdiff/2.2.2/diff.min.js"></script>
+
+<!-- Diffing UI Extension -->
+<script>
+
+  function DiffExtension(extensionsApi) {
+
+    extensionsApi.window.updateHeight();
+
+    // DOM elements.
+    var draftTextEl = document.querySelector('.draft-text');
+    var diffTextEl = document.querySelector('.diff-text');
+
+    // Currently published version of entry.
+    var publishedVersion = extensionsApi.entry.getSys().publishedVersion;
+
+    // Current locale
+    var fieldLocale = extensionsApi.field.locale;
+    var fieldId = extensionsApi.field.id;
+
+    var publishedValue;
+
+    // Populate the input field with the value of the entry.field.
+    draftTextEl.value = extensionsApi.entry.fields[extensionsApi.field.id].getValue();
+
+    extensionsApi.field.onValueChanged(onValueChanged);
+    extensionsApi.entry.onSysChanged(onSysChanged);
+    draftTextEl.addEventListener('input', keyboardInputHandler);
```

Comment:
`keyboardInputHandler` might be called before the `loadPublishedValue` resolves. Which means `publishedValue` will be `undefined`. This will cause `JsDiff.diffWordsWithSpace(publishedValue, draftTextEl.value);` on line `104` to throw.